src/libs/runtime-utils.js

```diff
@@ -32,13 +32,19 @@ export const defaultNumDataprocPreemptibleWorkers = 0
 export const defaultGpuType = 'nvidia-tesla-t4'
 export const defaultNumGpus = 1
 
+export const defaultLocation = 'US'
+export const defaultLocationType = 'multi-region'
+
+export const defaultComputeZone = 'US-CENTRAL1-A'
+export const defaultComputeRegion = 'US-CENTRAL1'
```

Comment:
Could you remind me why these had to be uppercase? I've made mistakes in the past by mixing up case and would love to reduce chances of that if possible.

Reply:
The value returned from Google's `storage/v1/b/${bucket}?fields=location` API returns locations in caps. There's a few places where we do comparisons with this value, so we make the default uppercase to match.